src/app/shared/hero/hero.component.css

```diff
@@ -36,7 +36,7 @@
 }
 .toolbar section {
   display: flex;
-  align-items: center;
```

Comment:
This does make the buttons look a bit funny on the Series page though (the toolbar is a shared component) -> http://cav.is/s/gystn.jpg